docs/pages/machine-id/workload-identity/getting-started.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Getting Started with Workload Identity
+description: Getting started with Teleport Workload Identity for SPIFFE and Machine ID
+---
+
+<Admonition type="tip" title="Preview">
+Teleport Workload Identity is currently in Preview. We are actively working on
+improving the feature and would love to [hear your feedback](mailto: product@goteleport.com).
+We currently do not recommend using this feature in production.
+</Admonition>
+
+Teleport's Workload Identity issues flexible short-lived identities intended
+for workloads. It is compatible with the industry-standard SPIFFE specification
+meaning that it can be used in place of other SPIFFE compatible identity
+providers.
+
+In this guide, you'll configure the RBAC necessary to allow a Bot to issue
+SPIFFE SVIDs and then configure `tbot` to expose a SPIFFE Workload API endpoint.
+You can then connect your workloads to this endpoint to receive SPIFFE SVIDs.
+
+## Getting Started
```

Comment:
I think we can remove this H2 and elevate "Prerequisites" to an H2, since this H2 has no other content.

Reply:
2285838b13